mnemonic-core/src/test/java/org/apache/mnemonic/Person.java

```diff
@@ -71,6 +75,13 @@ public abstract void setName(String name, boolean destroy)
       throws OutOfHybridMemory, RetrieveDurableEntityError;
 
   @DurableGetter
+  public abstract MemBufferHolder getNamebuffer();
```

Comment:
Could you please use Camel Style Naming convention e.g. getNameBuffer()/setNameBuffer ?